umbraco-cloud/set-up/private-nuget-feed.md

```diff
@@ -0,0 +1,74 @@
+
+# Private Nuget Feed
+
+A private NuGet feed is a package repository that is only accessible to a specific group of users, rather than being publicly available. Private feeds are often used to host internal libraries or proprietary software within an organization. NuGet is a package manager for the Microsoft development platform, including .NET. It makes it easy to add, remove, and update libraries and tools in Visual Studio projects.
+
+In this tutorial, we'll be covering how to set up a private NuGet feed with Umbraco Cloud. 
+
+## Set up
+
+To follow along with this tutorial, you'll need the following tools:
+
+1. Visual Studio
```

Comment:
Maybe link to the download page of visual studio: 
- https://visualstudio.microsoft.com/downloads/